audius-cli

```diff
@@ -407,17 +412,30 @@ def upgrade(launch_flag, manifests_path):
       - Call the launch function, this applies the config maps and restarts with the new configs.
     """
 
+    subprocess.run(["git", "fetch"], cwd=manifests_path)
+
     for file in [
         "audius/creator-node/creator-node-cm.yaml",
         "audius/discovery-provider/discovery-provider-cm.yaml",
         "audius/identity/identity-cm.yaml",
+        "audius/creator-node/creator-node-deploy-backend.yaml",
+        "audius/discovery-provider/discovery-provider-deploy.yaml",
+        "audius/discovery-provider/discovery-provider-deploy-no-workers.yaml",
+        "audius/identity/identity-deploy.yaml",
     ]:
         print(f"Reverting {file}")
         proc = subprocess.run(["git", "checkout", "--", file], cwd=manifests_path)
         if proc.returncode != 0:
             print(f"Unable to revert {file}", file=sys.stderr)
             sys.exit(1)
 
+    if branch:
+        print("Checking out to provided branch")
```

Comment:
should this just read "Checking out provided branch"

Reply:
yeah probably should. this is for auto upgrade so shouldn't be used by anyone atm. i'll fix it with a follow up PR

Reply:
ya nw